x-pack/plugins/monitoring/public/views/base_controller.js

```diff
@@ -129,11 +153,21 @@ export class MonitoringViewBaseController {
     // needed for chart pages
     this.onBrush = ({ xaxis }) => {
       const { to, from } = xaxis;
+      const fromTime = moment(from);
```

Comment:
I think `to` and `from` are unix timestamps but in `ms` instead of `s` and I think we might be able to get away with not using `moment` at all here - just doing the same comparison but in `ms`, then the additional below should be fairly straightforward without moment too. WDYT?

Reply:
You're right, but I don't think we actually need to send ms precision to the server, since it'll probably still get rounded to seconds somewhere down the line (assuming since we don't collect date at ms intervals (less than one second)). I think it might be a bit faster calc/sending seconds instead.

Also, I decided to keep the moment method here since that's what it was originally, even though you're right, it does take in strings according to `/kibana/src/legacy/ui/public/timefilter/time_history.d.ts`. Was just worried it might cause a regression somewhere and it's not really related to the issue.

That being said I have no problems applying these changes though, LMK

Reply:
>  I don't think we actually need to send ms precision to the server, since it'll probably still get rounded to seconds somewhere down the line (assuming since we don't collect date at ms intervals (less than one second)). I think it might be a bit faster calc/sending seconds instead.

Agreed.

My main point is around removing the `moment` conversion, as it seems unnecessary given what we're doing with the values. I think we do straight up division/subtraction to achieve the same result. I don't think we're dealing with any strings - just pure numbers. WDYT?

Reply:
I take it back actually. This is fine. No issue on my side!